editor/editor_node.cpp

```diff
@@ -5035,7 +5182,15 @@ void EditorNode::_load_open_scenes_from_config(Ref<ConfigFile> p_layout, const S
 	for (int i = 0; i < scenes.size(); i++) {
 		load_scene(scenes[i]);
 	}
-	save_layout();
+
+	// The canvas item editors clears the selection in set current scene for some reason!!!!
```

Comment:
```suggestion
	// The canvas item editor clears the selection in set current scene for some reason.
```
Although I'd check if this is still true, because there were some editor changes that could fix this.

Reply:
Ooops, this was actually a note to myself that should be removed